netlify/functions/render.js

```diff
@@ -51,28 +89,50 @@ const router = express.Router();
 router.get('/', async (req, res) => {
   const isBot = dectectBot(req.headers['user-agent']);
   const eventId = req.baseUrl.split('/')[2];
+
   if (isBot) {
-    const event = await getEvent(eventId)
-    const { data } = event
+    const event = await getEvent(eventId);
+
+    const { data } = event;
+
+    const xdai = await fulfillWithTimeLimit(1000, getxDaiTokens(eventId), null);
+
+    const main = await fulfillWithTimeLimit(1000, getMainnetTokens(eventId), null);
```

Comment:
I think it would be faster if you use Promise.all

`const [xdai, main] = Promise.all([fulfillWithTimeLimit(1000, getxDaiTokens(eventId), null), await fulfillWithTimeLimit(1000, getMainnetTokens(eventId), null)])`